build/templates/replace-tokens.yml

```diff
@@ -0,0 +1,29 @@
+steps:
+  - powershell: |
+      Get-ChildItem -Path ./src -Filter *.psd1 -Recurse |
+        % { Write-Host "Replace '0.0.0' to 'Package.Version' in $($_.FullName)"
+            $contents = Get-Content $_.FullName 
+            $contents = $contents -replace "ModuleVersion = '0.0.0'", "ModuleVersion = '#{Package.Version}#'"
```

Comment:
Should we only replace when it is `0.0.0`? Or just replace whatever the version is?

Reply:
Ah, you make a good point. Was first hesitate to do a regular expression check (like `\d\.\d\.\d`) bc it would also replace the versions in the `RequiredModules` section, but since we take in the `ModuleVersion` we could indeed do that. Nice catch! 👍